torch_geometric/transforms/__init__.py

```diff
@@ -19,6 +19,7 @@
 from .local_degree_profile import LocalDegreeProfile
 from .add_self_loops import AddSelfLoops
 from .remove_isolated_nodes import RemoveIsolatedNodes
+from .remove_duplicate_self_loops import RemoveDuplicateSelfLoops
```

Comment:
Can we move this to a separate PR? Not totally sure I get the purpose of this transform yet. Wouldn't `RemoveDuplicatedEdges` be sufficient?